examples/06_benchmarks/movielens.ipynb

```diff
@@ -72,28 +72,39 @@
    "execution_count": 1,
```

Comment:
<p>Why is it saying <code>CUDA version: No CUDA in this machine</code>? your machine should have CUDA installed. Maybe there is a problem with the function. Here you have something that it should work: <a href="https://github.com/miguelgfierro/pybase/blob/main/system/system_info.py" target="_blank">pybase/system_info.py at main · miguelgfierro/pybase (github.com)</a>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Let's create an issue about this